detox/detox.d.ts

```diff
@@ -1287,18 +1287,23 @@ declare global {
         interface NativeElementActions {
             /**
              * Simulate tap on an element
-             * @param point relative coordinates to the matched element (the element size could changes on different devices or even when changing the device font size)
+             * @param point coordinates in the element's coordinate space. Optional (default is the center of the element).
              * @example await element(by.id('tappable')).tap();
              * @example await element(by.id('tappable')).tap({ x:5, y:10 });
              */
             tap(point?: Point2D): Promise<void>;
 
             /**
              * Simulate long press on an element
-             * @param duration (iOS only) custom press duration time, in milliseconds. Optional (default is 1000ms).
+             * @param duration custom press duration time, in milliseconds. Optional (default is 1000ms).
```

Comment:
This is now different on iOS and Android, according to the platform. TODO: update.